package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@contentstack/delivery-sdk",
-  "version": "5.0.0",
+  "version": "5.1.0",
```

Comment:
@sunil-lakshman This will be 4.2.0. We are going to revert the PR of 5.0.0 most probably.